### DISCUSSION TITLE
[WASM] Uno0001 warning on DataPackageOperation.Move (drag & drop)

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#9594] [Q&A] [answered]
It's my understanding that drag & drop is [supported in principle ](https://github.com/unoplatform/uno/pull/4270).

However, drag&drop events like .DragEnter/.Drop/.DragItemsStarting require e.RequestOperation/e.AcceptedOperation/... being set, and those properties require the DataPackageOperation enum. Unfortunately the values of that enum throw Uno0001 warnings.

Is that expected?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. Not it's not, we'll fix that.

**Reply by DierkDroth** [reply]
Great. Thanks @jeromelaban
